Mathlib/CategoryTheory/Products/Unitor.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2024 Shanghe Chen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Shanghe Chen
+-/
+import Mathlib.CategoryTheory.Products.Basic
+import Mathlib.CategoryTheory.DiscreteCategory
+
+/-!
+# The left/right unitor equivalences `1 × C ≌ C` and `C × 1 ≌ C`.
+-/
+
+universe v u
+
+open CategoryTheory
+
+namespace CategoryTheory.prod
+
+variable (C : Type u) [Category.{v} C]
+
+/-- The left unitor functor `1 × C ⥤ C` -/
+@[simps]
+def leftUnitor : Discrete PUnit × C ⥤ C where
```

Comment:
As there is an implicit universe, it would be nice that it appears as the first, so that `universe v u` should be replaced by `universe w v u`, and the definitions changed as follows:
```suggestion
def leftUnitor : Discrete (PUnit : Type w) × C ⥤ C where
```

Reply:
I feel it does not matter very much.